tests/tests_C2_economic_functions.py

```diff
@@ -0,0 +1,100 @@
+import src.C2_economic_functions as e_functions
+
+project_life = 20
+wacc = 0.1
+investment_t0 = 220000
+tax = 0.15
+lifetime = [20, 15, 35]
+present_value = 295000
+crf = 0.12
+annuity = 35400
+annuity_factor = 1 / 0.12
+fuel_keys = {
+    "fuel_price": 1.3,
+    "fuel_price_change_annual": 0,
+    "project_lifetime": project_life,
+    "wacc": wacc
+}
+
+
+def test_annuity_factor():
+    """
+
+    Tests whether the MVS is correctly calculating the annuity factor
+    """
+    AF = e_functions.annuity_factor(project_life, wacc)
+    assert AF == 1 / 0.1 - 1 / (0.1 * (1 + 0.1) ** 20)
```

Comment:
```suggestion
    assert AF == 1 / discount_factor  - 1 / (discount_factor  * (1 +tax) ** project_life)
```
Please check if this is correct!